Customization for Patents Collection - Facet and Search Filter

AnushaPillai opened the pull request:
Resolved a subset of issue #374: Customized the facets and the search filter for the patent collections.

Modified the discovery file to add a new configuration for patent handles and added properties in the message.xml that defines the display title. The SimpleSearch property keys are for the search filter and the AdvancedSearch property keys are for facets.

Commit:
Customization for Patents Collection - Facet and Search Filter
Modified the discovery file to add a new configuration for patent handles and added properties in the message.xml that defines the display title. The SimpleSearch property keys are for the search filter and the AdvancedSearch property keys are for facets.

Commit:
Made changes to the handles referred in the discovery.xml file  as per production

alawvt commented:
Thanks, @AnushaPillai, this looks really good. The facets seem fine and all the search filters seem to work. 

I noticed these problems, which I'd like you to correct:  
- When I click on "More" in the Assignee facet, the title of the page is xmlui.Discovery.AbstractSearch.type_assignee
- When I click on "More" in the Assignee facet, the title of the page is xmlui.Discovery.AbstractSearch.type_inventor

You would probably want to create  a similar key for patenttype, in case we get more.

Commit:
Modified files to fix the reported issues and modified files to fix issue in results displayed using advance search(unknown author issue fix)
1.fix the label shown when "More" is clicked in the facet for all facets
2.search result view to display the inventor, rather than "unknown author."

alawvt commented:
Nice job! Patent Type, Inventor, Assignee, and Publication Date facets added along with many search filters.The inventor name displays correctly and not the assignee in search results. All "filter by" headings are correct.

alawvt commented:
@AnushaPillai , I have merged this branch. Be sure to pull vt_5_x_dev before continuing development (probably on a new branch) and eventually another pull request.
